src/views/invitation/Invitation.vue

```diff
@@ -0,0 +1,248 @@
+              </i18n-t>
+              <span
+                :class="`invitation__content__list__invit__text__invitation-status--${invit.status}`"
+              >
+                <template
+                  v-if="invit.status === INVITATION_VALIDATION_STATUS.ACCEPT"
+                >
+                  {{ $t("Invitation.invitAccepted") }}
+                </template>
+                <template
+                  v-else-if="invit.status === INVITATION_VALIDATION_STATUS.DENY"
+                >
+                  {{ $t("Invitation.invitDenied") }}
+                </template>
+                <template
+                  v-else-if="
+                    invit.status === INVITATION_VALIDATION_STATUS.PENDING
+                  "
+                >
+                  {{ $t("Invitation.invitPending") }}
+                </template>
+
+                ({{ $d(invit.created_at, "longMonth") }})
+              </span>
+            </div>
+            <div class="invitation__content__list__invit__button">
+              <template
+                v-if="invit.status === INVITATION_VALIDATION_STATUS.ACCEPT"
+              >
+                <AppLink
+                  :to="
+                    invit.project_id
+                      ? {
+                          name: routeNames.projectBoard,
+                          params: {
+                            spaceID: invit.cloud_id,
+                            projectID: invit.project_id
+                          }
+                        }
+                      : {
+                          name: routeNames.spaceBoard,
+                          params: {
+                            spaceID: invit.cloud_id
+                          }
+                        }
+                  "
+                >
+                  <BIMDataButton
+                    color="primary"
+                    :width="isMD ? 'inherit' : '130px'"
+                    fill
+                    radius
+                  >
+                    <template v-if="invit.project_name">
+                      {{
+                        $t(`Invitation.goToProject${isMD ? "Short" : "Long"}`)
+                      }}
+                    </template>
+                    <template v-else>
+                      {{ $t(`Invitation.goToSpace${isMD ? "Short" : "Long"}`) }}
+                    </template>
+                  </BIMDataButton>
+                </AppLink>
+              </template>
+              <template
+                v-if="invit.status === INVITATION_VALIDATION_STATUS.PENDING"
+              >
+                <div class="invitation__content__list__invit__button__pending">
+                  <BIMDataButton
+                    class="invitation__content__list__invit__button__pending--deny"
+                    width="35px"
+                    height="35px"
+                    radius
+                    ghost
+                    icon
+                    @click="onDenyInvitation(invit).then(fetchInvitations)"
```

Comment:
Handling async stuff in the template seems complex.
Do we want to call `fetchInvitations()` in the function `onDenyInvitation()` ?
@Kurtil